jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/J2735DirectionOfUse.java

```diff
@@ -16,7 +16,8 @@
 package us.dot.its.jpo.ode.plugin.j2735;
 
 public enum J2735DirectionOfUse {
-   FORWARD, 
-   REVERSE, 
-   BOTH
+   unavailable,
+   forward, 
+   reverse, 
+   both
```

Comment:
A goal/requirement is to be able to serialize and deserialize the ASN.1 XER format from asn1_codec.   Drew also stated that a goal of this PR is to produce output (JSON) closer to the standard.  That standard is ASN.1 JER.  Both XER and JER write ENUMERATED types like this one with the casing as defined in the ASN.1 specification document for the standard, so in this case, for example, an XER value would look like:
```xml
<forward/>
```
and the JER value would just be verbatim:
```json
"forward"
```
So that's a long-winded way of saying that yes we should care about the case of the strings that we write.

To comply with the Java style standard for Enums (which in general is a good thing that I agree with), we would also need to be able to deserialize the strings, so in addition to the `toString` we'd need to add a corresponding `fromString` to every enum, something like:

```java
public static J2735DirectionOfUse fromString(String direction) {
    return valueOf(direction.toUpperCase());
}
```

Furthermore, for Jackson to be able to deal with this enum, we would also have to add some annotations, like:

```java
    @Override
    @JsonValue
    public String toString() {
        return name().toLowerCase();
    }

    @JsonCreator
    public static J2735DirectionOfUse fromString(String direction) {
        return valueOf(direction.toUpperCase());
    }
```

For this enum, `toLowerCase` and `toUpperCase` work fine, but as you noted below, we'd need to define an additional property on the enum to deal with compound names.  In general, enum items in the J2735 standard are in hyphen-separated "kebab case".  For example, SPATs have a MovementPhaseState enum with items such as `permissive-Movement-Allowed`.  (I'm aware SPATs are out of scope of this PR; they are just the example I'm most familiar with that pops to mind.)   So the general pattern for enums would be:
```java
public enum MovementPhaseState {
   ...
   PERMISSIVE_MOVEMENT_ALLOWED("permissive-Movement-Allowed")
   ...
```
with the obvious changes to the toString and fromString methods.